.github/workflows/goTest.yml

```diff
@@ -38,17 +42,20 @@ jobs:
     outputs:
       matrix: ${{ steps.set-output.outputs.matrix }}
     steps:
-      - name: Set go-matrix latest version
+      -
+        name: Set go-matrix latest version
```

Comment:
This is ugly but ok.

Reply:
Eye of the beholder. I think it looks better.